x/profiles/types/error.go

```diff
@@ -7,16 +7,74 @@ import (
 )
 
 var (
-	ErrInvalidVersion      = sdkerrors.Register(ModuleName, 1, "invalid version")
+	// ErrInvalidVersion is returned if a wrong version of IBC channel is used
+	ErrInvalidVersion = sdkerrors.Register(ModuleName, 1, "invalid version")
+
+	// ErrMaxProfilesChannels is returned if the channel sequence exceed the max allowed
 	ErrMaxProfilesChannels = sdkerrors.Register(ModuleName, 2, "max profiles channels")
 
-	ErrProfileNotFound = sdkerrors.Register(ModuleName, 10, "profile not found")
+	// ErrProfileNotFound is returned when a profile doesn't exist
+	ErrProfileNotFound = sdkerrors.Register(ModuleName, 3, "profile not found")
+
+	// ErrProfileAlreadyCreated is returned when a profile has already been created
+	ErrProfileAlreadyCreated = sdkerrors.Register(ModuleName, 4, "profile already created")
+
+	// ErrInvalidNickname is returned when a profile's nickname doesn't respect the nickname parameters
+	ErrInvalidNickname = sdkerrors.Register(ModuleName, 5, "invalid profile nickname")
+
+	// ErrInvalidDTag is returned when a profile's DTag is empty or doesn't respect the DTag parameters
+	ErrInvalidDTag = sdkerrors.Register(ModuleName, 6, "invalid profile DTag")
+
+	// ErrInvalidBio is returned when a profile's Bio doesn't respect the Bio parameters
+	ErrInvalidBio = sdkerrors.Register(ModuleName, 7, "invalid profile biography")
+
+	// ErrInvalidDTagRequest is returned when a DTagRequest is not valid (doesn't exist or same sender/receiver)
+	ErrInvalidDTagRequest = sdkerrors.Register(ModuleName, 8, "invalid DTag transfer request")
+
+	// ErrBlockEqualUsers is returned when the users of a block are the same
+	ErrBlockEqualUsers = sdkerrors.Register(ModuleName, 9, "blocker and blocked cannot be the same user")
+
+	// ErrBlockUserAlreadyBlocked is returned if the blocked user has already been blocked
+	ErrBlockUserAlreadyBlocked = sdkerrors.Register(ModuleName, 10, "the user has already been blocked")
```

Comment:
Can we rename this something like `ErrDuplicatedBlock`? It would make it coherent with other errors as well